model/PermissionsService.php

```diff
@@ -256,8 +364,10 @@ private function triggerEvents(array $addRemove, string $resourceId, bool $isRec
         $this->eventManager->trigger(
             new DataAccessControlChangedEvent(
                 $resourceId,
+                $rootResourceId,
```

Comment:
- [ ] It is a breaking change. It should be the 4th parameter instead.

Reply:
Done: 4771810deb75922610b05aa381af98cb945e52f3